src/main/java/de/dataelementhub/model/handler/importhandler/StagedElementHandler.java

```diff
@@ -0,0 +1,243 @@
+    ElementType elementType = stagedElement.getIdentification().getElementType();
+    identification.setElementType(elementType);
+    ScopedIdentifier scopedIdentifier;
+    List<Member> members;
+    String convertedUrn = getUrnIfConvertedAndDraftStillAvailable(ctx,
+        importId, stagedElement.getValueDomainUrn());
+    if (!convertedUrn.equals("")) { // if element already exist return urn
+      return convertedUrn;
+    }
+    switch (elementType) {
+      case DATAELEMENT:
+        DataElement dataElement = new DataElement();
+        dataElement.setDefinitions(stagedElement.getDefinitions());
+        dataElement.setSlots(stagedElement.getSlots());
+        dataElement.setConceptAssociations(stagedElement.getConceptAssociations());
+        dataElement.setIdentification(identification);
+        String valueDomainUrn = getUrnIfConvertedAndDraftStillAvailable(ctx, importId,
+            stagedElement.getValueDomainUrn());
+        if (valueDomainUrn.equals("")) {
+          StagedElement memberAsStagedElement = getStagedElement(ctx, importId, userId,
+              stagedElement.getValueDomainUrn());
+          valueDomainUrn = stagedElementToElement(ctx, memberAsStagedElement,
+              namespaceUrn, userId, importId);
+        }
+        dataElement.setValueDomainUrn(valueDomainUrn);
+        scopedIdentifier = DataElementHandler.create(ctx, userId, dataElement);
+        markAsConverted(ctx, userId, importId, stagedElement.getIdentification().getUrn(),
+            scopedIdentifier);
+        return IdentificationHandler.toUrn(scopedIdentifier);
+      case DATAELEMENTGROUP:
+        DataElementGroup dataElementGroup = new DataElementGroup();
+        dataElementGroup.setDefinitions(stagedElement.getDefinitions());
+        dataElementGroup.setSlots(stagedElement.getSlots());
+        dataElementGroup.setIdentification(identification);
+        members =
+            handleMembers(ctx, importId, namespaceUrn, userId, stagedElement.getMembers());
+        dataElementGroup.setMembers(members);
+        scopedIdentifier = DataElementGroupHandler.create(ctx, userId, dataElementGroup);
+        markAsConverted(ctx, userId, importId, stagedElement.getIdentification().getUrn(),
+            scopedIdentifier);
+        return IdentificationHandler.toUrn(scopedIdentifier);
+      case RECORD:
+        Record record = new Record();
+        record.setDefinitions(stagedElement.getDefinitions());
+        record.setSlots(stagedElement.getSlots());
+        record.setIdentification(identification);
+        members =
+            handleMembers(ctx, importId, namespaceUrn, userId, stagedElement.getMembers());
+        record.setMembers(members);
+        scopedIdentifier = RecordHandler.create(ctx, userId, record);
+        markAsConverted(ctx, userId, importId, stagedElement.getIdentification().getUrn(),
+            scopedIdentifier);
+        return IdentificationHandler.toUrn(scopedIdentifier);
+      case ENUMERATED_VALUE_DOMAIN:
```

Comment:
Please comment on the fall-through here (see [google java codestyle](https://google.github.io/styleguide/javaguide.html#s4.8.4-switch))

Reply:
There is no need for a fall-through comment here because all blocks terminate abruptly using a return  statement

Reply:
`switch (input) {
  case 1:
  case 2:
    prepareOneOrTwo();
    // fall through
  case 3:
    handleOneTwoOrThree();
    break;
  default:
    handleLargeNumber(input);
}`

Notice that **no comment is needed** after case 1:, only at the end of the statement group.